package.json

```diff
@@ -40,10 +40,15 @@
     "dependencies": {
         "@dhis2/app-runtime": "^3.2.1",
         "@dhis2/prop-types": "^1.6.4",
-        "@dhis2/ui": "^7.2.7",
+        "@dhis2/ui": "^8.7.6",
```

Comment:
@Mohammer5 the change in the cypress test was required as a result of the major version bump in UI.